### ISSUE TITLE
Server-side synchronization problem

### ISSUE THREAD

**Issue by MateusKH** [Wakaba2/World-Renderer#13]
I need to solve the server synchronization problem, in this case the render only works on the host but not for the other players. Do you have any idea how to solve it?

**Comment by Wakaba2** [comment]
What are you trying to achieve with the rendering? That will change your approach.

**Comment by MateusKH** [comment]
the entire mod uses the world renderer. No fog, no sky, and no shape renderer.

**Comment by Wakaba2** [comment]
In multiplayer, if players are using the same mods, the renderer will work properly.

**Comment by Wakaba2** [comment]
Rendering requires to synchronize data between the client side and the server side. Global variables are helpful for rendering.

**Comment by MateusKH** [comment]
Yes, I am using a global variable. I tested it with a persistent player, session and map.
The map ended up giving this error
![image](https://github.com/user-attachments/assets/d45bc0af-76fb-4a0e-805c-6029f66d465d)

**Comment by Wakaba2** [comment]
It's probably not a World Renderer problem. Try regenerating the server world.
